specification/mediaservices/resource-manager/Microsoft.Media/Streaming/stable/2022-11-01/streamingservice.json

```diff
@@ -0,0 +1,2877 @@
+          "description": "The streaming endpoint sku name."
+        },
+        "capacity": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The streaming endpoint sku capacity."
+        }
+      },
+      "description": "The streaming endpoint current sku.",
+      "type": "object"
+    },
+    "ArmStreamingEndpointSku": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The streaming endpoint sku name."
+        }
+      },
+      "description": "The streaming endpoint sku.",
+      "type": "object"
+    },
+    "ArmStreamingEndpointCapacity": {
+      "properties": {
+        "scaleType": {
+          "type": "string",
+          "readOnly": true
+        },
+        "default": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The streaming endpoint default capacity."
+        },
+        "minimum": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The streaming endpoint minimum capacity."
+        },
+        "maximum": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The streaming endpoint maximum capacity."
+        }
+      },
+      "description": "The streaming endpoint sku capacity.",
+      "type": "object"
+    },
+    "ArmStreamingEndpointSkuInfo": {
+      "properties": {
+        "resourceType": {
+          "type": "string"
+        },
+        "capacity": {
+          "$ref": "#/definitions/ArmStreamingEndpointCapacity"
+        },
+        "sku": {
+          "$ref": "#/definitions/ArmStreamingEndpointSku"
+        }
+      },
+      "type": "object"
+    },
+    "StreamingEndpointSkuInfoListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ArmStreamingEndpointSkuInfo"
+          },
+          "x-ms-identifiers": [],
+          "description": "The result of the List StreamingEndpoint skus."
+        }
+      },
+      "type": "object"
+    },
+    "AsyncOperationResult": {
+      "properties": {
+        "error": {
+          "description": "The error object",
+          "$ref": "../../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorDetail"
+        },
+        "name": {
+          "type": "string",
+          "description": "Operation Id of the async operation."
+        },
+        "status": {
+          "description": "Operation status of the async operation.",
+          "type": "string",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "InProgress"
+          ],
+          "x-ms-enum": {
+            "name": "AsyncOperationStatus",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "The status of an async operation.",
+      "type": "object"
+    },
+    "IngestInterruption": {
```

Comment:
It would be good to prefix each of these types with "LiveEvent", e.g., "LiveEventIngestInterruption". This makes it easier to work out how the various types are used when using the client library.